### DISCUSSION TITLE
skoala/quickstart/offline-upgrade/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1495] [Announcements]
# skoala/quickstart/offline-upgrade/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/quickstart/offline-upgrade/

**Comment by liyuerich** [comment]
添加微服务引擎的 helm 仓库。heml repo add skoala-release http://{harbor url}/chartrepo/{project} 

命令行中的  helm 写错了， 写成了 heml

**Reply by samzong** [reply]
感谢反馈，已提交修复反馈。

**Reply by Michelle951** [reply]
fixed：
<img width="758" alt="image" src="https://github.com/DaoCloud/DaoCloud-docs/assets/35519562/b61e387b-b444-4f7d-b4fd-50e015ac6589">
